src/utils/apolloClient.ts

```diff
@@ -1,12 +1,27 @@
 import { InMemoryCache } from 'apollo-cache-inmemory'
 import { ApolloClient } from 'apollo-client'
 import { BatchHttpLink } from 'apollo-link-batch-http'
+import { notNullable } from './nullables'
 
-export const GIRAFFE_ENDPOINT =
+let realGiraffeEndpoint: string
```

Comment:
move this into a func and we can avoid the let maybe? 👯‍♂️

Reply:
sure 🙌